changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1 | 2020-03-23
+* Drop matplotlib < 3 requirement.
```

Comment:
Style 🚓: new line after section, `matplotlib` package name in backticks.

Reply:
👍